<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoolFM Recoder (KBS Radio 89.1Mhz)</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>녹화 설정</string>
   </property>
   <widget class="QTimeEdit" name="timeEdit_StartTm">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>시작시각</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>파일이름</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit_EndTm">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
    <property name="time">
     <time>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>종료시각</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FileName">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>시작</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>진행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>중지</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Open">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>보라</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_Schedule">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>10</y>
     <width>60</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>방송스케쥴</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditLogger">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>491</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_Schedule</tabstop>
  <tabstop>timeEdit_StartTm</tabstop>
  <tabstop>timeEdit_EndTm</tabstop>
  <tabstop>checkBox_Open</tabstop>
  <tabstop>lineEdit_FileName</tabstop>
  <tabstop>pushButton_Start</tabstop>
  <tabstop>pushButton_Stop</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_Schedule</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>setRecInfo(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Start</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>tryRecording()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Stop</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>stopRecording()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setRecInfo(int)</slot>
  <slot>tryRecording()</slot>
  <slot>stopRecording()</slot>
 </slots>
</ui>
